css/mainStyle.css

```diff
@@ -0,0 +1,264 @@
+margin-top: 2px;
+}
+
+.dashboard>ul{
+width: 100%;
+}
+
+#listItemDashboardMain.lbl-dashboard-main{
+padding:32px 0px 12px 0px;
+color: white;
+font-size: 20px;
+margin: 0px;
+border-bottom: 0px white;
+}
+
+#listItemDashboardCampaignTab.lbl-dashboard-campaign-tab{
+width: 97%;
+background-color:white;
+border-right: 6px solid #d83c3c;
+border-bottom: 0px white;
+text-align: center;
+padding: 12px 0px 7px 0px;
+color: #c5c3c3;
+margin:0px;
+}
+
+.lbl-dashboard{
+margin:1px;
+color: #c5c3c3;
+border-bottom: 1px solid white;
+text-align: center;
+width: 100%;
+padding: 10px 0px 7px 0px;
+}
+
+/*right side bar starts here*/
+
+.main-bar{
+flex-direction:column;
+width: 740px;
+background-color: white;
+margin-left: 4px;
+}
+
+.breadcrumb-bar{
+padding-top: 12px;
+width: 725px;
+}
+
+.lbl-mainbar-dashboard{
+color:#13a0f8;
+text-shadow: 0px 1px 1px gray;
+}
+
+.lbl-mainbar-campaigns{
+	color:#d83c3c;
+}
+
+.campaign-list{
+width: 725px;
+background-color: white;
+}
+
+.campaign-label-row{
+width: 100%;
+background-color: white;
+padding-bottom: 20px;
+padding-top: 45px;
+}
+
+.lbl-recent-campaign{
+float: left;
+color: #13a0f8;
+text-shadow: 0px 1px 1px gray;
+}
+
+.lbl-showall{
+float: right;
+color:#666666;
+border:none;
+background-color: white;
+outline:none;
+padding: 0px 0px 0px 5px;
+}
+
+.lbl-update{
+float: right;
+color:#666666;
+border-style: none;
+border-right: 1px solid #666666;
+background-color: white;
+outline:none;
+padding: 0px 5px 0px 0px;
+}
+
+.lbl-mainbar-dashboard:after{
+color: #666666;
+font-weight: bold;
+font-size: 20px;
+content:'>';
+padding-left: 6px;
+}
+
+.campaign-field-row{
+height: 32px;
+width: 725px;
+background-color: #d83c3c;
+padding: 4px;
+border-radius: 4px;
+}
+
+.campaign-field-list{
+padding-top: 9px;
+}
+
+.campaign-field-list:after {
+content: "";
+display: block;
+clear: both;
+overflow: hidden;
+}
+
+.campaign-field-list>li{
+color: white;
+list-style: none;
+float:left;
+}
+
+.campaign-data-table{
+overflow: auto;
+height: 461px;
+width: 735px;
+background-color: #b4b4b4;
+margin-top: 10px;
+border-radius: 4px;
+}
+
+.campaign-data-table>ul{
+display: flex;
+flex-direction:row;
+justify-content:space-between;
+list-style: none;
+color: #666666;
+margin-top: 8px;
+padding:5px 14px 12px 14px;
+border-bottom: 1px solid #8C8B8C;
+}
+
+.campaign-field-row>ul>li:nth-child(1){
+padding-left: 11px;
+text-shadow: -1px 1px 1px rgba(0,0,0,0.5);
+}
+.campaign-field-row>ul>li:nth-child(2){
+padding-left: 125px;
+text-shadow: -1px 1px 1px rgba(0,0,0,0.5);
+}
+.campaign-field-row>ul>li:nth-child(3){
+padding-left: 52px;
+text-shadow: -1px 1px 1px rgba(0,0,0,0.5);
+}
+.campaign-field-row>ul>li:nth-child(4){
+padding-left: 27px;
+text-shadow: -1px 1px 1px rgba(0,0,0,0.5);
+}
+.campaign-field-row>ul>li:nth-child(5){
+padding-left: 26px;
+text-shadow: -1px 1px 1px rgba(0,0,0,0.5);
+}
+.campaign-field-row>ul>li:nth-child(6){
+padding-left: 83px;
+text-shadow: -1px 1px 1px rgba(0,0,0,0.5);
+}
```

Comment:
This looks little odd, why we need to specify styling for every column? This could be achieved by HTML restructuring.
And css will be more readable if properties have little padding form left, like -
.campaign-field-list:after {
    content: "";
    display: block;
    clear: both;
    overflow: hidden;
}
instead of ->

.campaign-field-list:after {
content: "";
display: block;
clear: both;
overflow: hidden;
}